Facebook.Android/source/Xamarin.Facebook/Transforms/Metadata.xml

```diff
@@ -33,4 +34,7 @@
     <attr path="/api/package[@name='com.facebook.share.internal']/class[@name='VideoUploader.FinishUploadWorkItem']/method[@name='getParameters' and count(parameter)=0]" name="visibility">protected</attr>
     <attr path="/api/package[@name='com.facebook.share.internal']/class[@name='VideoUploader.StartUploadWorkItem']/method[@name='getParameters' and count(parameter)=0]" name="visibility">protected</attr>
 
+    <remove-node path="/api/package[contains(@name, 'com.facebook.share.internal')]" />
+    <remove-node path="/api/package[@name='com.facebook.login.widget']/class[@name='DeviceLoginButton.DeviceLoginClickListener']" />
+    <remove-node path="/api/package[@name='com.facebook.login']/class[@name='CustomTabLoginMethodHandler']" />
```

Comment:
CustomTabLoginMethodHandler is listed in their docs, but without any explanation and I can't seem to find where it would be publicly used, so I think it's safe to remove.  The DeviceLoginButton.DeviceLoginClickListener isn't even in the docs, and looks to also be safe to remove.